test/unit/puppetlabs/services/ca/certificate_authority_core_test.clj

```diff
@@ -483,6 +513,20 @@
         (is (true? (fs/exists? signed-cert-path)))
         (is (= 204 (:status response))))))
```

Comment:
This is maybe outside the scope of this PR, but it seems kind of bizarre that this test is still using `/production/certificate_status` while the other tests in this file are using `/v1/certificate_status`.